cloudformation/adx-data-plane.cfn.yaml

```diff
@@ -0,0 +1,245 @@
+Description: Create S3 bucket and cross-account role for ADX data plane with minimal permissions.
```

Comment:
If we do a change on the CDK code, would we have to copy the generated CFN template and manually put the updates on this template? Is there a way for us to do something in maybe Github workflows or something that does:
* run CDK
* take the output of the generated CFN yaml and put it in S3
and that way we don't really need to have this in the repo? (avoiding the manual step and the risk of having them out of sync)

Reply:
ah yes - I like this idea a lot. I'll remove the CFN template from source and deploy the synthesized template to S3 in the deploy workflow